docs/source/contribution-guidelines.rst

```diff
@@ -2,5 +2,5 @@
 Contribution Guidelines
 =======================
 
-.. todo:: Guidelines about how to contribute
```

Comment:
Why is this being removed?

Reply:
The `..todo::` was not allowing the intergration with `readthedocs` at that time I was desperate to make it work and was removing everything. It was giving some error

Reply:
What error is being emitted?

Reply:
I really don't remember. I think once we get the `parse_argparse.py` cleaned by I can put the `.. todo::` back and we can correct it then